src/random.rs

```diff
@@ -1,7 +1,12 @@
-use rand::{self, Rng};
-use super::Cmd;
+extern crate bytebot_rpn as rpn;
```

Comment:
`extern crate` should generally only be in the toplevel file, not in each module. You probably want `use rpn;`